### PR TITLE
Rocket implement

### PR DESCRIPTION
**Pull request by TaiwoEnoch**
## Create this PR to feature the below listed tasks ⬇️ 

- [ ] [[2pt] Display joined missions in My profile - Filters]
- [ ] [[4pt] Fetch rockets - Fetch data]
- [ ] Press enter to move this card
- [ ] [[4pt] Display rockets - Lists render]
- [ ] Press enter to move this card
- [ ] [[1pt] Switch badges for Rockets - Conditional components]
- [ ] [4pt] Fetch rockets - Fetch data
- [ ] [4pt] Display rockets - Lists render
- [ ] [1pt] Switch badges for Rockets - Conditional components
- [ ] [2pt] Display reserved rockets in My
- [ ] [1pt] Implement rocket booking cancelation - Actions
- [ ] [[3pt] Implement rocket booking - Actions]

### CODE REVIEW

**Review by dev-mamun — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @TaiwoEnoch :wave: , 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Highly organized codes :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/routes/RocketItems.js
Comment (dev-mamun) [inline src/components/routes/RocketItems.js:1]: The design does match. Please try to make a design like a sample.

image

image

Comment (TaiwoEnoch) [reply src/components/routes/RocketItems.js:1]: Noted, thank you for your feedback.

**Review by dev-mamun — COMMENTED** [review]
## STATUS: APPROVED 🟢

## Hi @TaiwoEnoch 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉

- Nice UI design. heavy_check_mark
- Professional README file. heavy_check_mark
- Gitflow is followed. :heavy_check_mark:
- No linter errors. heavy_check_mark
- PR has a good title and summary. ✔️


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dev-mamun — COMMENTED** [review]
Approved

**Pull request merged**